### PR TITLE
feat: generate proguard rule file for android

### PR DESCRIPTION
**Pull request by amrbashir**
This PR requires https://github.com/tauri-apps/wry/pull/926

### What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Docs
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### Checklist
- [x] This PR will resolve https://github.com/tauri-apps/tauri/issues/6694
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/wry/blob/dev/.changes/readme.md).
- [x] I have added a convincing reason for adding this feature, if necessary
- [x] It can be built on all targets and pass CI/CD.

### CODE REVIEW

**Comment by lok52** [comment]
Hi! After including the suggested proguard file, I am getting another error.

```
04-16 14:58:12.683 18445 18445 E AndroidRuntime: java.lang.NoSuchFieldError: no "Lbencher/tauri_bencher/RustWebView;" field "m_webview" in class "Lbencher/tauri_bencher/MainActivity;" or its superclasses
04-16 14:58:12.683 18445 18445 E AndroidRuntime: 	at android.os.MessageQueue.nativePollOnce(Native Method)
04-16 14:58:12.683 18445 18445 E AndroidRuntime: 	at android.os.MessageQueue.next(MessageQueue.java:335)
04-16 14:58:12.683 18445 18445 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:161)
04-16 14:58:12.683 18445 18445 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:288)
04-16 14:58:12.683 18445 18445 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7872)
04-16 14:58:12.683 18445 18445 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
04-16 14:58:12.683 18445 18445 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
04-16 14:58:12.683 18445 18445 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
04-16 14:58:12.688   586  1994 E ActivityManager: App crashed on incremental package bencher.tauri_bencher which is 100% loaded.
```

File: src/webview/android/kotlin/proguard-wry.pro [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright 2020-2023 Tauri Programme within The Commons Conservancy
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-License-Identifier: MIT
+
+-keep class {{package}}.* {
+  native <methods>;
+}
+
+-keep class {{package}}.WryActivity {
```
Comment (lok52) [inline src/webview/android/kotlin/proguard-wry.pro:9]:
```suggestion
-keep class {{package}}. TauriActivity {
```

After this change everything works fine

Comment (amrbashir) [reply src/webview/android/kotlin/proguard-wry.pro:9]: This PR is pending for https://github.com/tauri-apps/wry/pull/926 to be merged where `TauriActivity` will be changed to `WryActivity`

Comment (wusyong) [reply src/webview/android/kotlin/proguard-wry.pro:9]: You can update it now.

**Review by wusyong — APPROVED** [review]

**Pull request merged**